lib/BaseSendGridClientInterface.php

```diff
@@ -55,11 +55,13 @@ public function __construct($auth, $host, $options = array())
      * Make an API request.
      *
      * @param Mail $email A Mail object, containing the request object
+     * @param array|null $query A query, containing additional settings,
+     *                          e.g., css_inline
      *
      * @return Response
      */
-    public function send(Mail $email)
+    public function send(Mail $email, array $query = null)
```

Comment:
```suggestion
    public function send(Mail $email, ?array $query = null)
```
needs a `?` for PHP 8.4 compatibility